torch/_inductor/ir.py

```diff
@@ -4670,13 +4683,8 @@ def codegen(self, wrapper):
                     # repeat_interleave(const at::Tensor & repeats, c10::optional<int64_t> output_size=c10::nullopt)
```

Comment:
nit: should these comments be removed as they are moved into `get_aten_cpp_kernel_name` ?